src/platform/packages/private/kbn-grid-layout/grid/grid_layout.test.tsx

```diff
@@ -34,13 +35,21 @@ const renderGridLayout = (propsOverrides: Partial<GridLayoutProps> = {}) => {
     ...propsOverrides,
   } as GridLayoutProps;
 
-  const { rerender, ...rtlRest } = render(<GridLayout {...props} />);
+  const { rerender, ...rtlRest } = render(
+    <EuiThemeProvider>
+      <GridLayout {...props} />
+    </EuiThemeProvider>
+  );
 
   return {
     ...rtlRest,
     rerender: (overrides: Partial<GridLayoutProps>) => {
       const newProps = { ...props, ...overrides } as GridLayoutProps;
-      return rerender(<GridLayout {...newProps} />);
+      return rerender(
+        <EuiThemeProvider>
+          <GridLayout {...newProps} />
+        </EuiThemeProvider>
+      );
```

Comment:
I think you can do this to avoid passing the provider for rerender:
```
 const { rerender, ...rtlRest } = render(<GridLayout {...props} />, { wrapper: EuiThemeProvider });

  return {
    ...rtlRest,
    rerender: (overrides: Partial<GridLayoutProps>) => {
      const newProps = { ...props, ...overrides } as GridLayoutProps;
      return rerender(<GridLayout {...newProps} />);
    },
  };
```

Reply:
Nice 🔥 Done in 18f090d67ce62e8005ce3aff663daf46d7c6939b